src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+    <App />,
+  document.getElementById('root')
+);
+
+reportWebVitals();
```

Comment:
Please format your code. Always finish your editing by running the formatter.

Reply:
i ran the formatter. it made the render() in to one line witch might not make it more readable but at the moment the index.js do not have much code so i guess it's still easy to read